src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Super class of address, email and phone
+ * contain two common attributes namely value and isPrivate
+ * @author benwong
+ *
+ */
+public class Contact {
+	
+	public final String value;
```

Comment:
try to make it private. in general, start off with private members, then use protected or public only if absolutely necessary.